README.rst

```diff
@@ -189,6 +191,17 @@ In the ``sphinx-docs`` folder, use ``make docs`` to build the HTML documents
 and static assets, add it to the docs folder, and commit it for publication on
 Github Pages. After the build completes, push to GitHub from the ``docs`` folder.
 
+Troubleshooting
+-------------
+
+Trouble launching Solr? Try configuring Java to version 8. Later versions of 
+Java will cause Solr 6 to time out.
+
+Solr changes not reflected in search results? ``solrconfig.xml`` must be 
+updated in Solr's main directory: ``solr/server/solr/[CORE]/conf/solrconfig.xml``
+
+We currently do not support Python 3.7 or Solr 7 and above.
+
```

Comment:
These are good things to document, but I'm not sure they belong in the readme. Let's create a DEVELOPERNOTES.rst document with this info.

Reply:
Another thought would be to pull all of the installation and setup info into a separate file from the readme, and then a troubleshooting section would be perfectly appropriate. It seems like it might already be long enough to warrant that, what do you think?

Reply:
I'm OK with keeping all of this in a README, but don't have strong opinions. We do list the supported version of python at the top with other versions; maybe we could just add "Java 8" to that list.

Reply:
I've moved Troubleshooting to a DEVELOPERNOTES.rst document and I've added Java and Solr versioning to the top of README.rst

Reply:
Agree - python version is documented at the top of the readme; please add solr version and note the java version in reference to Solr. 

While you're updating that line, would you correct the Django version? We're on 2.2 in develop now but I didn't update that yet.